cmd/vic-machine/create/create.go

```diff
@@ -1091,7 +1091,12 @@ func (c *Create) generateCertificates(server bool, client bool) ([]byte, *certif
 
 	var certs []byte
 	// generate the certs and keys with names conforming the default the docker client expects
-	err := os.MkdirAll(c.certPath, 0700)
+	files, err := ioutil.ReadDir(c.certPath)
+	if len(files) > 0 {
```

Comment:
hmm... why should we do this? Feels like if user wants to generate certificate, that could be put into whatever directories